pkgs/by-name/ko/kontemplate/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildGoModule
+, fetchgit
+}:
+buildGoModule {
+  pname = "kontemplate";
+  version = "1.8.0-unstable-2024-06-09";
+
+  src = fetchgit {
+    url = "https://code.tvl.fyi/depot.git";
+    sha256 = "sha256-H1HpguUorvOD9/2KfC52+dp30/8Sx4cjPQdyFgNw5Hw=";
+    rev = "fa29d7344e2e631455f3759736234f083d741f71";
+  };
+
+  patches = [
+    # Migrates the project to Go modules.
+    ./0001-chore-kontemplate-Migrate-to-Go-modules.patch
+  ];
+
+  postPatch = ''
+    ls -la ops/kontemplate
+  '';
+
+  modRoot = "ops/kontemplate";
+
+  vendorHash = "sha256-xPGVM2dq5fAVOiuodOXhDm3v3k+ncNLhlk6aCtF5S9E=";
+
+  meta = {
+    description = "Extremely simple Kubernetes resource templates";
+    mainProgram = "kontemplate";
+    homepage = "https://code.tvl.fyi/about/ops/kontemplate";
+    downloadPage = "https://github.com/tazjin/kontemplate/releases";
```

Comment:
That's not true anymore, right?

Reply:
The Download Page?

Reply:
I think the `downloadPage` can be removed here, there's no current page where I publish releases anymore.